### PR TITLE
Abuse make_traitor proc

### PR DESCRIPTION
**Pull request by eswordthecat**
honk

### CODE REVIEW

File: code/game/gamemodes/steal_items.dm [hunk 1]
```diff
@@ -35,30 +35,25 @@
...
 
 		if (isliving(owner.current))
 			all_items += get_contents_in_object(owner.current)
-		//testing("Got [all_items.len] objects in [owner.current]")
 
-		if(areas && areas.len)
-			for (var/area_type in areas)
-				all_items += get_contents_in_object(locate(area_type), /obj)
-		//testing("Starting to check the list for [typepath]")
+		for (var/area_type in areas)
```
Comment (eswordthecat) [inline code/game/gamemodes/steal_items.dm]: There's no need for if(areas && areas.len) unless it's the area of var you're using in for loop

File: code/datums/mind.dm [hunk 2]
```diff
@@ -1083,13 +1079,23 @@ datum/mind
 			A.icon_state = "ai-malf"
 
 	proc/make_Tratior()
```
Comment (ComicIronic) [inline code/datums/mind.dm]: pls no

File: code/game/gamemodes/traitor/traitor.dm [hunk 3]
```diff
@@ -162,7 +162,7 @@
 
 
 /datum/game_mode/proc/greet_traitor(var/datum/mind/traitor)
-	traitor.current << "<SPAN CLASS='danger'><CENTER><BIG>You are a traitor!</BIG></CENTER></SPAN>"
+	traitor.current << "<SPAN CLASS='danger'><CENTER><BIG>You are now a traitor!</BIG></CENTER></SPAN>"
```
Comment (ComicIronic) [inline code/game/gamemodes/traitor/traitor.dm]: Span this, it's used more than once.

Comment (eswordthecat) [inline code/game/gamemodes/traitor/traitor.dm]: What?

File: code/datums/mind.dm (hunk 2 above)
Comment (eswordthecat) [inline code/datums/mind.dm]: Don't you want to be a Tratior bb?

File: code/datums/mind.dm [hunk 4]
```diff
@@ -840,12 +840,8 @@ datum/mind
 							A.set_zeroth_law("")
 							A.show_laws()
 				if ("traitor")
-					if(!(src in ticker.mode.traitors))
+					if (make_Tratior())
```
Comment (clusterfack) [inline code/datums/mind.dm]: Typo

File: code/datums/mind.dm (hunk 2 above)
Comment (ComicIronic) [inline code/datums/mind.dm]: Rename proc or I suicide

File: code/game/gamemodes/traitor/traitor.dm (hunk 3 above)
Comment (ComicIronic) [inline code/game/gamemodes/traitor/traitor.dm]: Make it a span class, for standards.

**Comment by eswordthecat** [comment]
Alright final changes done, can be pulled.

File: code/stylesheet.dm [hunk 5]
```diff
@@ -62,6 +62,10 @@ h1.alert, h2.alert		{color: #000000;}
 .rough					{font-family: 'Trebuchet MS', cursive, sans-serif;}
 .say_quote				{font-family: Georgia, Verdana, sans-serif;}
 .interface				{color: #330033;}
+.big					{font-size: 150%}
+.bold					{font-weight: bold;}
```
Comment (ririchiyo) [inline code/stylesheet.dm]: There's no need for this as this should be done through <b> or <strong> unless you're wanting to make it bolder or boldest.

File: code/stylesheet.dm [hunk 6]
```diff
@@ -62,6 +62,10 @@ h1.alert, h2.alert		{color: #000000;}
 .rough					{font-family: 'Trebuchet MS', cursive, sans-serif;}
 .say_quote				{font-family: Georgia, Verdana, sans-serif;}
 .interface				{color: #330033;}
+.big					{font-size: 150%}
```
Comment (ririchiyo) [inline code/stylesheet.dm]: You forgot to add a semicolon for this line.

File: code/datums/mind.dm [hunk 7]
```diff
@@ -1082,14 +1081,27 @@ datum/mind
 			special_role = "malfunction"
 			A.icon_state = "ai-malf"
 
-	proc/make_Tratior()
-		if(!(src in ticker.mode.traitors))
+	proc/make_traitor()
```
Comment (ririchiyo) [inline code/datums/mind.dm]: Please fix this file up so they're absolute paths instead of relative paths.

**Comment by ririchiyo** [comment]
Please stop merging upstream into your feature branch that no-one else will use. Please learn how to rebase.

**Comment by eswordthecat** [comment]
Haloooooooooooooooooooooooooooooo

**Pull request merged**